src/test/java/suite/optout/OptoutTest.java

```diff
@@ -99,25 +101,88 @@ public void testV2LogoutWithV2TokenGenerate(String label, Operator operator, Str
             "suite.optout.TestData#optoutTokenEmailArgs",
             "suite.optout.TestData#optoutTokenPhoneArgs"
     })
+    @Order(4)
     public void testV2LogoutWithV2TokenGenerateOldParticipant(String label, Operator operator, String operatorName, String type, String identity) throws Exception {
         IdentityTokens generateResponse = operator.v2TokenGenerate(type, identity, true).getIdentity();
         Thread.sleep(OPTOUT_DELAY_MS);
         JsonNode logoutResponse = operator.v2TokenLogout(type, identity);
         assertThat(logoutResponse).isEqualTo(Mapper.OBJECT_MAPPER.readTree("{\"body\":{\"optout\":\"OK\"},\"status\":\"success\"}"));
         addToken(
                 "old participant " + label,
-                operatorName,
+                operator,
                 "v2",
                 "v2",
                 generateResponse.getRefreshToken(),
                 Mapper.OBJECT_MAPPER.readTree(generateResponse.getJsonString()).at("/refresh_response_key").asText()
         );
     }
 
-    private void addToken(String label, String operatorName, String tokenGenerateVersion, String tokenLogoutVersion, String refreshToken, String refreshResponseKey) {
-        outputs.add("List.of(\"%s\", \"%s\", \"%s\", \"%s\", \"%s\", \"%s\"),%n".formatted(
-                        label, operatorName, tokenGenerateVersion, tokenLogoutVersion, refreshToken, refreshResponseKey
-                )
-        );
+    @Order(5)
+    @ParameterizedTest(name = "/v0/token/refresh after {3} generate and {4} logout - {0} - {2}")
+    @MethodSource({
+            "newRefreshTokenArgs"
+    })
+    public void testV0TokenRefreshAfterOptOut(String label, Operator operator, String tokenGenerateVersion, String tokenLogoutVersion, String refreshToken, String refreshResponseKey) throws Exception {
+        assumeThat(tokenGenerateVersion).isEqualTo("v0");
+
+        JsonNode response = this.waitForOptOutResponse(operator::v0CheckedTokenRefresh, refreshToken, "{\"advertisement_token\":\"\",\"advertising_token\":\"\",\"refresh_token\":\"\"}");
+
+        assertThat(response).isEqualTo(Mapper.OBJECT_MAPPER.readTree("{\"advertisement_token\":\"\",\"advertising_token\":\"\",\"refresh_token\":\"\"}"));
+    }
+
+    @Order(6)
+    @ParameterizedTest(name = "/v1/token/refresh after {3} generate and {4} logout - {0} - {2}")
```

Comment:
Since the parameter `String operatorName` became `Operator operator`, that might affect how the name is displayed here at placeholder `{2}`

The same applies to the other methods whose operator name parameter changed

Reply:
Updated